quest/create-choice.js

```diff
@@ -1,4 +1,21 @@
 // build create choice function, render 3 radio inputs
+export function createChoice(choice){
+    const label = document.getElementById('create-choice');
```

Comment:
it would be better to create a label for each radio button and append the labels to a div.